docker-compose.yml

```diff
@@ -0,0 +1,8 @@
+version: "3.7"
+services:
+  cloudflare-ddns:
+    image: ghcr.io/akarys42/cloudflare-ddns-docker
+    container_name: ddns_client
+    enviroment:
```

Comment:
tested using `env_file: .env` instead of `environment:` and works as intended. Just add a `.env` file to the same directory.

ref: [env_file | Compose specification | Docker Documentation](https://docs.docker.com/compose/compose-file/#env_file)